internal/grpcapi/order_service.go

```diff
@@ -2,32 +2,75 @@ package grpcapi
 
 import (
 	"context"
+	"database/sql"
+	"fmt"
 
-	pb2 "github.com/Sugar-pack/orders-manager/pkg/pb"
 	"github.com/Sugar-pack/users-manager/pkg/logging"
 	"github.com/google/uuid"
 	"github.com/jmoiron/sqlx"
+
+	"github.com/Sugar-pack/orders-manager/internal/db"
+
+	"github.com/Sugar-pack/orders-manager/pkg/pb"
 )
 
 type OrderService struct {
-	pb2.OrdersManagerServiceServer
+	pb.OrdersManagerServiceServer
 	dbConn *sqlx.DB
 }
 
-func (s OrderService) InsertOrder(cxt context.Context, order *pb2.Order) (*pb2.OrderTnxResponse, error) {
-	logger := logging.FromContext(cxt)
+func (s OrderService) InsertOrder(ctx context.Context, order *pb.Order) (*pb.OrderTnxResponse, error) {
+	logger := logging.FromContext(ctx)
 	logger.Info("ReceiveOrder")
 	orderID := uuid.New()
 	txID := uuid.New()
-	query := "INSERT INTO orders ( id,  user_id, label, created_at ) VALUES ($1, $2, $3, $4)"
-	_, err := s.dbConn.Exec(query, orderID.String(), order.UserId, order.Label, order.CreatedAt.AsTime())
+	parseUserID, err := uuid.Parse(order.UserId)
+	if err != nil {
+		logger.Error("Error parsing user id ", err)
+
+		return nil, fmt.Errorf("error parsing user id %w", err)
```

Comment:
grpc error should contain error-code.
take a look https://jbrandhorst.com/post/grpc-errors/

Reply:
done